### PR TITLE
Lured Pokemon now appear on map

### PR DESCRIPTION
**Pull request by hgentry**
## Description

Lured pokemon are now visible on map. No more "missing" pokemon near lures!
## How Has This Been Tested?

I've been testing this pretty heavily for a while now. The only "issue" is that sometimes a lure will appear on the map with no lured Pokemon. This happens because the lured pokemon's spawn has expired and been cleaned away, but the search has not yet re-reached that pokestop to find out what new Pokemon appeared.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: static/map.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ var selectedStyle = 'light';
 var map_pokemons = {}
 var map_gyms = {}
 var map_pokestops = {}
+map_lure_pokemons = {}
```
Comment (andrewda) [inline static/map.js]: Declare this with `var`

File: static/map.js [hunk 2]
```diff
@@ -381,24 +399,24 @@ function clearStaleMarkers() {
...
 function updateMap() {
 
-    localStorage.showPokemon = localStorage.showPokemon || true;
-    localStorage.showGyms = localStorage.showGyms || true;
-    localStorage.showPokestops = localStorage.showPokestops || false;
-    localStorage.showScanned = localStorage.showScanned || false;
+    loadPokemon = localStorage.showPokemon || true;
+    loadGyms = localStorage.showGyms || true;
+    loadPokestops = true;
+    loadScanned = localStorage.showScanned || false;
```
Comment (andrewda) [inline static/map.js]: First time using these - should be declared with `var`.

**Comment by JZ6** [comment]
good stuff +1

**Comment by andrewda** [comment]
:+1:

File: config/credentials.json [hunk 3]
```diff
@@ -1,3 +0,0 @@
-{
```
Comment (invisiblek) [inline config/credentials.json]: deleting this file?

File: static/map.js [hunk 4]
```diff
@@ -14,6 +14,7 @@ var $selectNotify = $("#notify-pokemon");
 
 var language = document.documentElement.lang == "" ? "en" : document.documentElement.lang;
 var idToPokemon = {};
+//google.maps.Marker.prototype.pokemon_id = -1;
```
Comment (invisiblek) [inline static/map.js]: just remove this?

**Comment by AHAAAAAAA** [comment]
+1, rebase

**Comment by sparkycus123** [comment]
hi guys. newbie here. how can i add this on the map program?

**Comment by andrewda** [comment]
A couple issues to this PR are fixed in #1669

**Comment by Bitti09** [comment]
Any possibilitiy to mark the lured Pokem,ons in the popup  info Like 
`
Zubat - #41 (Lured)`
?

File: static/map.js [hunk 5]
```diff
@@ -275,18 +280,22 @@ function setupGymMarker(item) {
...
         map: map,
+        zIndex: 2,
+        optimized: false, 
         icon: 'static/forts/' + imagename + '.png',
     });
 
+
     marker.infoWindow = new google.maps.InfoWindow({
-        content: pokestopLabel(!!item.lure_expiration, item.last_modified, item.active_pokemon_id, item.latitude, item.longitude)
+        content: pokestopLabel(!!item.lure_expiration, item.last_modified, item.active_pokemon_id, item.latitude +.003, item.longitude+ .003)
```
Comment (vbevan) [inline static/map.js]: @hgentry Why was + 0.003 added to the lat and long? It throws off the actual position when trying to map out pokestops, unless I'm missing something?

**Pull request merged**